server/ai_mediaserver.go

```diff
@@ -721,7 +721,19 @@ func processStream(ctx context.Context, params aiRequestParams, req worker.GenLi
 			if err == nil {
 				err = errors.New("unknown swap reason")
 			}
-			params.liveParams.sendErrorEvent(err)
+			// report the swap
+			monitor.SendQueueEventAsync("stream_trace", map[string]interface{}{
+				"type":        "orchestrator_swap",
+				"stream_id":   params.liveParams.streamID,
+				"request_id":  params.liveParams.requestID,
+				"pipeline":    params.liveParams.pipeline,
+				"pipeline_id": params.liveParams.pipelineID,
+				"message":     err.Error(),
+				"orchestrator_info": map[string]interface{}{
```

Comment:
is this the orchestrator being swapped AWAY from or TO?

Reply:
away from